docs/guides/1.2.1-upgrade-guide.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: "couchbase-capella"
+page_title: "Couchbase Capella Provider 1.2.0: Upgrade and Information Guide"
+sidebar_current: "docs-couchbase-capella-guides-120-upgrade-guide"
+description: |-
+Couchbase Capella Provider 1.2.0: Upgrade and Information Guide
+---
+
+# Couchbase Capella Provider 1.2.1: Upgrade and Information Guide
+
+## Bug Fixed
+
+* Export capella audit logs with [`couchbase-capella_audit_log_export`](https://registry.terraform.io/providers/couchbasecloud/couchbase-capella/latest/docs/resources/audit_log_export) was failing with the v1.2.0 release, which has been rectified in the v1.2.1 release.
+
+## Changes
```

Comment:
Should there also be an update to https://github.com/couchbasecloud/terraform-provider-couchbase-capella/blob/main/CHANGELOG.md ?

Reply:
We generally create changelog pr, after release documentation get merged